src/main-process/atom-window.js

```diff
@@ -152,6 +151,16 @@ class AtomWindow extends EventEmitter {
     return paths.every(p => this.containsPath(p))
   }
 
+  loadDataOverProcessBoundary () {
+    Object.defineProperty(this.browserWindow, 'loadSettingsJSON', {
+      get: () => JSON.stringify(Object.assign({
+        userSettings: this.atomApplication.configFile.get(),
+        projectSettings: this.projectSettings
+      }, this.loadSettings)),
+      configurable: true
+    })
+  }
```

Comment:
The name `loadDataOver...` makes it sound like this method itself is going to load or retrieve some data, when really it just defines an accessor method that allows the window process to retrieve data later. It's hard to give this weird little code path a name; I might just leave the code un-abstracted in the constructor.

Another though - should we just define this getter using `get` syntax as a method? I can't remember why it needs to be configurable.

Reply:
You're definitely right, I can just move it back. 

I think we changed it to configurable because Nuclide previously had to monkey-patch the property (https://github.com/atom/atom/pull/16701). We've since (within the last few days) fixed this so removing it sounds good to me!